commands/BarracksCommand.js

```diff
@@ -79,16 +79,21 @@ class BarracksCommand {
     return this.userConfiguration.saveAuthenticationToken(token);
   }
 
+  cleanupProgramOptions(program) {
```

Comment:
Please create a wrapper of the commander module with all the global configuration for the command.